scripts/system_tests_utils.ts

```diff
@@ -56,6 +59,35 @@ const accounts: {
 	},
 ];
 
+export const createNewAccount = async (config?: {
+	unlock?: boolean;
+	refill?: boolean;
+}): Promise<{ address: string; privateKey: string }> => {
+	const acc = createAccount();
+	const clientUrl = getSystemTestProvider().replace('ws://', 'http://');
+	if (config?.unlock) {
+		const web3Personal = new EthPersonal(clientUrl);
+		await web3Personal.importRawKey(
+			getSystemTestBackend() === 'geth' ? acc.privateKey.slice(2) : acc.privateKey,
+			'123456',
+		);
+		await web3Personal.unlockAccount(acc.address, '123456', 500);
```

Comment:
You might want to unlock the account for a bit longer, 1000?

Reply:
yep you are right